configs/components/curl.rb

```diff
@@ -27,6 +27,7 @@
   elsif platform.is_windows?
     pkg.build_requires "runtime-#{settings[:runtime_project]}"
     pkg.environment "PATH", "$(shell cygpath -u #{settings[:gcc_bindir]}):$(PATH)"
+    pkg.environment "NM" , "/usr/bin/nm" if platform.name =~ /windowsfips-2016/
```

Comment:
I'm not sure what the implications are of using Cygwin `nm` with binaries built by mingw... I assume this problem would go away if we were using a newer version of mingw tool chain?

Reply:
I was able to find 13.2.0 in the default repository(https://chocolatey.org/api/v2) of choco: https://community.chocolatey.org/packages/mingw
I think this is the latest version. It ships with nm 2.39. Unfortunately, it still results in the `global_symbol_pipe` variable being empty with this version.
The system nm (`/usr/bin/nm`) is version 2.42 which doesn't result the `global_symbol_pipe` issue. So, I manually set it to system nm.

I also tried with mingw 8.1.0 (nm version 2.25) which was available in the default repo, that also fails with the same error.